lib/rules/eol-last.js

```diff
@@ -1,62 +1,95 @@
 /**
- * @fileoverview Require file to end with single newline.
+ * @fileoverview Require or disallow file to end with single newline.
  * @author Nodeca Team <https://github.com/nodeca>
+ * @author kdex <https://github.com/kdex>
```

Comment:
No need to apologize! Looks like some cases where we missed it, and should probably clean them up at some point.